projects/krasnor/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gp-5-remesh",
+  "version": "1.0.0",
+  "description": "Geometry Processing",
+  "main": "src/main.js",
+  "scripts": {
+    "start": "webpack-dev-server --host localhost --open",
+    "build": "webpack --env.prod"
+  },
+  "author": "Changkun Ou",
+  "license": "GPL-3.0",
```

Comment:
Maybe more information about GPLv3 that might be good to know for your future open source work:

```
GNU GPLv3:
1. Anyone can copy, modify and distribute this software.
2. You have to include the license and copyright notice with each and every distribution.
3. You can use this software privately.
4. You can use this software for commercial purposes.
5. If you dare build your business solely from this code, you risk open-sourcing the whole code base.
6. If you modify it, you have to indicate changes made to the code.
7. Any modifications of this code base MUST be distributed with the same license, GPLv3.
8. This software is provided without warranty.
9. The software author or license can not be held liable for any damages inflicted by the software.
```

Full: https://github.com/mimuc/gp-ws2021/blob/master/LICENSE